packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' as ui show ParagraphStyle, TextStyle, StrutStyle, lerpDouble, S
 
 import 'package:flutter/foundation.dart';
 
+import '../../rendering.dart' show kDefaultEllipsis;
```

Comment:
I don't think we should import something from the rendering library to painting, as the rendering library is one level higher above architecture-wise (see: https://flutter.dev/images/arch-overview/archdiagram.png). 

Could you move the constant to this file?